source/Calamari.Azure/Integration/Security/ServicePrincipal.cs

```diff
@@ -9,7 +9,7 @@ public static string GetAuthorizationToken(string tenantId, string applicationId
             var authContext = GetContextUri(activeDirectoryEndPoint, tenantId);
             Log.Verbose($"Authentication Context: {authContext}");
             var context = new AuthenticationContext(authContext);
-            var result = context.AcquireToken(managementEndPoint, new ClientCredential(applicationId, password));
+            var result = context.AcquireTokenAsync(managementEndPoint, new ClientCredential(applicationId, password)).GetAwaiter().GetResult();
```

Comment:
It looks like the sync version is no longer available in the latest version and since I have upgraded this dependency, https://github.com/OctopusDeploy/Calamari/pull/192/files#diff-5c2422bfd230f086caac361ea9cfa9a2R49, the code changed slightly, do u want me to go back ?